nemoguardrails/testing/test_chat.py

```
@@ -0,0 +1,202 @@
+from nemoguardrails import LLMRails, RailsConfig
+from nemoguardrails.testing.fake_model import FakeLLMModel
+from nemoguardrails.utils import new_event_dict, new_uuid
+
+_TEST_PROVIDERS_WITH_TOKEN_USAGE = ["openai", "azure_openai", "nim"]
+
+
+def _is_data_in_events(events: List[Dict[str, Any]], event_data: List[Dict[str, Any]]) -> bool:
+    if len(events) != len(event_data):
+        return False
+
+    for event, data in zip(events, event_data):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)